README.md

```diff
@@ -1 +1,186 @@
+  </ul>
+</details>
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Added HTML
+- Added CSS
+- Added Linters
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+### Prerequisites
+
+1. A Browser (Preferably Google Chrome)
+2. A Code Editor
+3. Internet Connection
+4. Git
+
+<!-- SETUP -->
+### Setup
+
+Clone this repository to your desired folder using:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Cheptoo21/Portfolio-setup-and-mobile-first.git
+```
+<!-- INSTALL -->
+
+### Install
+
+Install this project with the latest version of Git and a code editor.
+
+### Usage
+
+In order to run the project open index.html with live server.
+
+### Deployment
+
+You can deploy this project using:
+1.  A Code Editor
+2. Latest version of Git
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Cheptoo Patience**
+
+- GitHub: [@Cheptoo21](https://github.com/Cheptoo21)
+- Twitter: [@CheptooGyts](https://twitter.com/CheptooGyts)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/cheptoo-patience-73499015a)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **My Projects**
+- [ ] **My Languages Frameworks**
+- [ ] **My Contacts**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTION -->
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!--SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please give it some starts ⭐️⭐️⭐️⭐️⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Special acknowledgement to [@Omar-Muhamad](https://github.com/Omar-Muhamad) and [@microverseinc](https://github.com/microverseinc)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQS -->
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **How were the Linters utilised?**
+
+  - The Linters were utilised with the help of resources provided by [@microverseinc](https://github.com/microverseinc).
+
+- **What new features should be expected in the next release of the project?**
+
+  - We are currently working on adding **My Projects**, **My Languages Frameworks**, and **My Contacts**.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Your license file link is currently broken it shows a 404 error when clicked please kindly follow this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to setup your license file correctly :+1: :sparkles: